.azure/pipelines/jobs/default-build.yml

```diff
@@ -101,6 +101,9 @@ jobs:
       PB_AssetRootUrl: ''
       PB_RestoreSource: ''
       PB_PublishBlobFeedKey: ''
+      AccessTokenSuffix: ''
+    ${{ if ne(variables['System.TeamProject'], 'public') }}:
+      AccessTokenSuffix: '$(dotnetclimsrc-read-sas-token)'
```

Comment:
Why are both `$(AccessTokenSuffix)` and `$(PB_ACCESSTOKENSUFFIX)` needed, especially containing the same value❔

Reply:
I've been testing stuff out in internal builds so the .yml files are getting a bit ugly, I'll clean everything up once I get stuff working